packages/cli/src/utils/generate.ts

```diff
@@ -82,15 +83,39 @@ function copyUserSrcToCustomizations() {
 
 async function copyTheme() {
   const storeConfig = await import(userStoreConfigFileDir)
-
-  try {
-    copyFileSync(
-      `${userThemesFileDir}/${storeConfig.theme}.scss`,
-      tmpThemesCustomizationsFileDir
+  if (storeConfig.theme) {
+    const customTheme = `${userThemesFileDir}/${storeConfig.theme}.scss`
+    if (existsSync(customTheme)) {
+      try {
+        copyFileSync(customTheme, tmpThemesCustomizationsFileDir)
+        console.log(
+          `${chalk.green('success')} - ${
+            storeConfig.theme
+          } theme has been applied`
+        )
+      } catch (err) {
+        console.error(`${chalk.red('error')} - ${err}`)
+      }
+    } else {
+      // TODO: add link to our doc about creating a custom theme on faststore evergreen
```

Comment:
TODO:  review this [doc section](https://www.faststore.dev/how-to-guides/theming/creating-a-theme).